back/engines/free/frontend/app/services/frontend/url_service.rb

```diff
@@ -24,7 +24,7 @@ def model_to_path(model_instance)
       when User
         subroute = 'profile'
         slug = model_instance.slug
-      when Comment, OfficialFeedback ### comments and official feedbacks do not have a path yet, we return the post path for now
+      when Comment, InternalComment, OfficialFeedback ### comments and official feedbacks do not have a path yet, we return the post path for now
```

Comment:
This mean CTA button on mails do not, in fact, link to the related internal_comment, though the button copy says it does.
This functionality will be added in a later PR (see [CL-3881](https://citizenlab.atlassian.net/browse/CL-3881) description for more details)